protostar-rust/src/lib.rs

```diff
@@ -63,6 +88,8 @@ fn internal_collect_tests(
     linked_libraries: Option<&Vec<LinkedLibrary>>,
     test_files: Vec<Utf8PathBuf>,
     corelib_path: Option<&Utf8PathBuf>,
+    test_name_filter: Option<&str>,
+    exact_match: bool,
```

Comment:
I was considering this, but I though that maybe passing only actually required arguments to internal methods would be better? This could discourage doing some weird hacks, unlike when whole config is available. 

But I see your point, not really sure what would be a better approach here.